README.md

```diff
@@ -1,3 +1,23 @@
-# Setup
+# Hosted Link
 
-Create .env.development and .evn.test files in the root of this repo. .env.developmet should each set PGDATABASE to be whatever databases are used in ./db/setup.sql respectively. (currently nc_news for development, and nc_news_test for test)
+Link to hosted server: https://news-server-k1ka.onrender.com/api
+
+# Summary
+
+This project is for setting up the back end of a new-esque server. It uses postgresql to create and manage databases via requests made to the app from express.
+
+# Setup Instructions
+
+Clone this repository into your local machine by copying the link provided by the green code button and using the git clone command, or by copying it from here: "git clone https://github.com/MHWatchy/news-server-project.git". Do not include the speech marks and double check where in your file system you are cloning this repo. 
+The dependencies to be installed include: dev(husky, jest, jest-sorted, supertest), dotenv, express, pg, pg-format. This can be done manually or with the following command: "npm install -d husky jest jest-sorted supertest && npm install dotenv, express, pg, pg-format". Do not include the speech marks and double check you are within the cloned repo.
```

Comment:
Can summarise all this by saying `run npm install`

It will install all the dependencies in the package.json - so no need to install these all like this